config/environments/development.rb

```diff
@@ -38,4 +38,6 @@
 
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
+
+  config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.